pyof/v0x04/asynchronous/error_msg.py

```diff
@@ -477,7 +477,7 @@ def __init__(self, xid=None, exp_type=None, experimenter=None, data=b''):
         super().__init__(xid)
         self.exp_type = exp_type
         self.experimenter = experimenter
-        self.data = data
+        self.data = b'' if data is None else data
```

Comment:
`BinaryData()` -> `b''`; it's bad having GenericTypes when they are not needed.
The cleaner way is to set `b''` directly in the `__init__` method - same behavior, better readability.